src/Microsoft.Identity.Client/Internal/Requests/InteractiveRequest.cs

```diff
@@ -58,21 +63,22 @@ internal class InteractiveRequest : RequestBase
             IWebUI webUi)
             : base(serviceBundle, authenticationRequestParameters, interactiveParameters)
         {
+            _authenticationRequestParameters = authenticationRequestParameters;
```

Comment:
This is passed to the base object. Should be available from there?

Reply:
yep. thanks. missed that.

---
In reply to: [258170835](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/881#discussion_r258170835) [](ancestors = 258170835)